assets/stylesheets/screens/_main.scss

```diff
@@ -0,0 +1,44 @@
+body.main.get {
```

Comment:
I just move some sample styles from `application` file to the specific screen file.
I'll remove remove out when working with the dashboard screen.